query/src/sessions/session.rs

```diff
@@ -63,16 +65,38 @@ impl Session {
         let session_settings = Settings::try_create(&conf)?;
         let ref_count = Arc::new(AtomicUsize::new(0));
         let status = Arc::new(Default::default());
+        match typ {
+            SessionType::MySQL => {
+                let mysql_conn_id = session_mgr
```

Comment:
It's incorrect, we must save the mysql_conn_id in `fetch_add`

Reply:
saved in there: self.mysql_basic_conn_id.fetch_add(1, Ordering::Relaxed);

Maybe we can directly use session_mgr.mysql_basic_conn_id?

Reply:
Must save the return value of `fetch_add`, see https://doc.rust-lang.org/std/sync/atomic/struct.AtomicU32.html#method.fetch_add

Otherwise, you can't ensure the load value is not changed.